src/App.vue

```diff
@@ -798,25 +798,27 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     // if filing is not a draft, proceed with payment
     if (!isDraft && filingComplete) {
       this.setIsFilingPaying(true)
-      const paymentToken = filingComplete?.header?.paymentToken
-      if (paymentToken) {
+      const paymentToken = filingComplete.header?.paymentToken
+      const filingId = filingComplete.header?.filingId
```

Comment:
Here we have to get the Filing ID from the filing, since it could be newly created and this is the first time we know its Filing ID. (If this was already a draft then we would already know its Filing ID.)